blog/2019/01/fluxdiffeq.md

```diff
@@ -618,7 +618,7 @@ Instead, what we are learning is the tiny ODE system from which the ODE
 solution is generated. I.e., the neural network inside the neural_ode
 layer learns this function:
 
-```math
+```text
```

Comment:
what happened to tex support?

Reply:
https://github.com/JuliaLang/www.julialang.org/pull/2000#issuecomment-1872987041